packages/http/src/types.ts

```diff
@@ -16,7 +16,7 @@ export type OperationDetails = HttpOperation;
 
 export type HttpVerb = "get" | "put" | "post" | "patch" | "delete" | "head";
 
-export interface ServiceAuthentication {
+export interface Authentication {
```

Comment:
```suggestion
/** @deprecated use Authentication */
type ServiceAuthentication = Authentication;
export interface Authentication {
```

This unfortunately would be a breaking change might need to add

Reply:
Done